packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -671,6 +671,15 @@ class _FocusState extends State<Focus> {
     Widget child = widget.child;
     if (widget.includeSemantics) {
       child = Semantics(
+        // Automatically request the focus for a focusable widget when it
+        // receives an input focus action from the semantics. Nothing is needed
+        // for losing the focus because if focus is lost, that means another
+        // node will gain focus and take focus from this widget.
+        // TODO(gspencergoog): Allow this to be set on iOS once the issue is debugged.
+        // Issue: https://
+        onFocus: defaultTargetPlatform != TargetPlatform.iOS && _couldRequestFocus
```

Comment:
Does this issue reproduce on Android? I'm not sure if it's iOS-only, or if there's only a test on iOS so that's where it was caught.

Reply:
As far as I can tell, no, but as you say, there are no tests on Android that test this. The failure mode seems iOS specific (I doubt Android would implement their text field identically to iOS to the point where it has a similar effect on the bounding box of the native text field), but I suppose something different could happen on Android.

I could also exclude Android explicitly, however, if you think that it's likely enough to worry about.